Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -132,6 +132,16 @@ public async Task<SeriesViewModel> PostSeriesAsync(SeriesUploadViewModel newSeri
             }
 
             await CreateSeriesTeamMappingAsync(teams, series.ID);
+
+            //assign default series surface (To Be Decided)
+            await _context.SeriesSurface.AddAsync(
+                new SeriesSurface { 
+                    SeriesID = series.ID,
+                    SurfaceID = 0 
+                }
+            );
+            await _context.SaveChangesAsync();
+
```

Comment:
Would it make more sense to move this up to line 112, so you don't have to save changes twice and the Series and it's mapping to default Surface are created in the same transcation?

Reply:
I always thought that `series.ID` was propagated after `_context.Series` was saved to SQL. Is this not the case?